managers/elasticsearch.py

```diff
@@ -29,14 +29,32 @@ def createElasticConnection(self, scheme=None, host=None, port=None, user=None,
 
         creds = '{}:{}@'.format(user, pswd) if user and pswd else ''
 
-        host = '{}://{}{}:{}'.format(scheme, creds, host, port)
+        multHosts = []
+
+        if isinstance(host, list) or \
+            isinstance(host, dict) or \
+            isinstance(host, set) or \
+            isinstance(host, tuple):
+            
+            for i in host:
+                multHosts.append('{}://{}{}:{}'.format(scheme, creds, i, port))
+            
+            connectionConfig = {
+                'hosts': multHosts,
+                'timeout': timeout,
+                'retry_on_timeout': True,
+                'max_retries': 3
+            }
 
-        connectionConfig = {
-            'hosts': [host],
-            'timeout': timeout,
-            'retry_on_timeout': True,
-            'max_retries': 3
-        }
+        else:
+            host = '{}://{}{}:{}'.format(scheme, creds, host, port)
+
+            connectionConfig = {
+                'hosts': [host],
+                'timeout': timeout,
+                'retry_on_timeout': True,
+                'max_retries': 3
+            }
```

Comment:
opportunity to make this DRY-er, just add this to the existing `multHosts` array and then have only one `connectionConfig` block, makes it easier to maintain in the future if we want to update this config!